data_dir/scylla-dash-per-server-nemesis.master.json

```diff
@@ -6516,8 +6516,7 @@
     "schemaVersion": 31,
     "style": "dark",
     "tags": [
-      "master",
-      "longevity-twcs-3h"
```

Comment:
We had 2 tags here - "master" and "$test_name". Are you sure we should delete the second one?

Reply:
I think we can leave master tag, because we always install as master and don't differentiate by version. Additional tags will be added. 
@dkropachev what do you think?

Reply:
Ok, if it is intentional and no expected drawbacks then le it be so.